.github/workflows/config.yml

```diff
@@ -83,8 +68,46 @@ jobs:
           ./foreman/run_tests.sh --exclude-tag=salmon --exclude-tag=transcriptome --exclude-tag=affymetrix
           .github/scripts/upload_test_coverage.sh foreman
 
-      - name: Push built docker images
```

Comment:
do we still need to push docker images after main tests and after downloader/no-op tests?

Reply:
This is a good question, although main tests are gone now. I'm not actually 100% sure whether we do or not. Not sure why this came up, or why pulling the cache seems to fill almost all of our disk space.

I've made an issue to investigate that: https://github.com/AlexsLemonade/refinebio/issues/2692